### DISCUSSION TITLE
Wireguard Update

### DISCUSSION THREAD

**Discussion by DHYTFG** [complexorganizations/wireguard-manager#188] [Q&A]
Hello. I installed a Debian script on the vps
wireguard-manager. How do I update wireguard itself? I didn't find an item for updating. When trying to update via sudo apt install, wireguard writes that 30 packages could not be updated. Is it possible to update via a script or in some other way?

**Comment by ghost** [comment]
```
apt-get update && apt-get upgrade -y && apt-get dist-upgrade -y 
```

Try this.
